CONTRIBUTING.md

```diff
@@ -73,9 +73,8 @@ encouraged to try other DVC repositories- especially real-world cases!
 
 <a id='warning'></a>
 
-> **Warning**
-> When using any project that relies on an isolated Python environment (`conda`,
-> `venv`, etc.), Microsoft's
+> **Warning** When using any project that relies on an isolated Python
```

Comment:
Missing `:`

Reply:
Darn. The linter is now removing perfectly valid markdown new lines (2 trailing spaces) 🤷🏼

Reply:
p.s. but only on CI, not from pre-commit Git hook.